### DISCUSSION TITLE
How well does PrimeQA compare to FAISS?

### DISCUSSION THREAD

**Discussion by francisco-becker** [primeqa/primeqa#480] [General]
Hello,

My team and I are exploring PrimeQA as an alternative to FAISS, however we are not sure how well it would work, and if it would even out perfom FAISS, do you have any benchmarks? Could anyone try to answer this question?

Thank you

**Comment by avisil** [comment]
PrimeQA neural retrievers e.g DPR, ColBERT both use FAISS under the covers so you should be fine.
